storage/src/testFixtures/java/tech/pegasys/artemis/storage/StubStorageUpdateChannel.java

```diff
@@ -13,13 +13,19 @@
 
 package tech.pegasys.artemis.storage;
 
+import java.util.Optional;
 import tech.pegasys.artemis.storage.api.StorageUpdateChannel;
 import tech.pegasys.artemis.storage.events.diskupdates.StorageUpdate;
 import tech.pegasys.artemis.storage.events.diskupdates.StorageUpdateResult;
 import tech.pegasys.artemis.util.async.SafeFuture;
 
 public class StubStorageUpdateChannel implements StorageUpdateChannel {
 
+  @Override
+  public SafeFuture<Optional<Store>> onStoreRequest() {
+    return null;
```

Comment:
nit: Return failed future instead of `null`.

Reply:
Done